libraries/AP_Networking/AP_Networking_port.cpp

```diff
@@ -353,9 +353,11 @@ bool AP_Networking::Port::send_receive(void)
         // handle outgoing packets
         uint32_t available = writebuffer->available();
         available = MIN(300U, available);
+#if HAL_GCS_ENABLED
```

Comment:
This'll do.

I should probably try to get my `AP_MAVLINK_ENABLED` branch working at some stage...